packages/suite/src/views/wallet/coinmarket/common/CoinmarketTransactions/CoinmarketTransaction/CoinmarketTransactionId.tsx

```diff
@@ -0,0 +1,28 @@
+import { Row } from '@trezor/components';
+import { spacingsPx, typographyStylesBase } from '@trezor/theme';
+import { Translation } from 'src/components/suite';
+import { CoinmarketTransactionText } from 'src/views/wallet/coinmarket';
+import styled from 'styled-components';
+
+interface CoinmarketTransactionIdProps {
+    transactionId: string;
+}
+
+const CoinmarketTradeIdTitle = styled.span`
+    padding-right: ${spacingsPx.xxs};
+`;
+
+const CoinmarketTradeId = styled.span`
+    font-weight: ${typographyStylesBase.highlight.fontWeight};
```

Comment:
Use Text

Reply:
It is not possible to use label with semiBold or am I missing something?

Reply:
I guess the point is more like, if it's not in typographyStyles, don't use it:D

Reply:
Deleted then!